adobe_vipm/flows/fulfillment/transfer.py

```diff
@@ -172,18 +172,18 @@ def _check_transfer(mpt_client, order, membership_id):
         ],
         key=itemgetter(0),
     )
-
-    order_lines = sorted(
-        [(line["item"]["externalIds"]["vendor"], line["quantity"]) for line in order["lines"]],
-        key=itemgetter(0),
-    )
-    if adobe_lines != order_lines:
-        error = ERR_MEMBERSHIP_ITEMS_DONT_MATCH.to_dict(
-            lines=",".join([line[0] for line in adobe_lines]),
+    if adobe_lines:
```

Comment:
I'd like to use the guard clauses pattern to avoid nested code -> https://refactoring.guru/es/replace-nested-conditional-with-guard-clauses

```
if not adobe_lines:
    return True

order_lines = ...
if adobe_lines == order_lines:
    return True

error = ....
return False
...
```